server/runtime.go

```diff
@@ -1131,6 +1145,10 @@ func NewRuntime(logger, startupLogger *zap.Logger, db *sql.DB, jsonpbMarshaler *
 		allAfterReqFunctions.afterAuthenticateFacebookFunction = goAfterReqFunctions.afterAuthenticateFacebookFunction
 		startupLogger.Info("Registered Go runtime After function invocation", zap.String("id", "authenticatefacebook"))
 	}
+	if goAfterReqFunctions.afterAuthenticateFacebookInstantGameFunction != nil {
```

Comment:
Also add support for Lua after hook, see: https://github.com/heroiclabs/nakama/blob/master/server/runtime_lua.go#L691

Reply:
Done